static/app/views/replays/detail/focusTabs.tsx

```diff
@@ -16,23 +17,35 @@ const ReplayTabs: Record<TabKey, string> = {
 type Props = {};
 
 function FocusTabs({}: Props) {
+  const {location} = useRouteContext();
```

Comment:
Could use `useLocation()` (which wraps `useRouteContext`... not sure which is preferred, maybe grep and see what's more used).